### DISCUSSION TITLE
When ever i stash  my changes it gets deleted

### DISCUSSION THREAD

**Discussion by sourav-biggboss** [community/community#33567] [Other Feature Feedback, Questions, & Ideas]
I want to stash  my code and it gets deleted on git desktop simply i select all the files and i then click on stash changes

**Comment by airtower-luna** [comment]
That's what stashing is supposed to do. It stores your current changes on a stack (that's the "stash") and then resets your files to the latest commit. You can apply the changes again later (though I'm not sure what the button/menu entry is called in GitHub Desktop). See also: [Git Tools - Stashing and Cleaning](https://git-scm.com/book/en/v2/Git-Tools-Stashing-and-Cleaning)
